manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: "en-US"
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer
+PackageURL: "https://www.fjsoft.at/en/"
```

Comment:
```suggestion
PackageUrl: "https://www.fjsoft.at/en/"
```

Reply:
I copied the field from the docs... Yaml should already be fixed and I opened a Issue: https://github.com/MicrosoftDocs/windows-uwp/issues/3167

Reply:
Daaamn you're right, i knew i've seen PackageURL before and i've typed it myself at times, good catch 👍